pkg/cmd/triggerbinding/describe_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
Could you please add a test case for a triggerbinding with multiple params?

Reply:
Added Thank you :+1:

Reply:
Just one last thing: Could one of the params be an Array instead of a String?

Reply:
There is no helper function implemented to call in order to add Array